controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -703,60 +703,28 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(const std::string &sr
                                                            ieee1905_1::CmduMessageRx &cmdu_rx)
 {
     LOG(DEBUG) << "Received AP_AUTOCONFIGURATION_WSC_MESSAGE";
+    // According to section 17.1.3 in the Multi-AP Specification version 1.0,
+    // if an AP-Autoconfiguration WSC message is sent by the Multi-AP Agent,
+    // it includes one AP Radio basic capabilities TLV and one WSC TLV (containing M1).
+    // If the the message is sent by the Multi-AP controller, it includes one
+    // AP radio Identifier TLV and one or more WSC TLVs (containing M2).
+    // Check if this is the case and this is an M2 message we sent to the agent
+    // which was just looped back (when we send the M2 to the local agent,
+    // it will be published back on the local bus because the destination is our
+    // AL-MAC, and the controller does listen to this CMDU message type).
+    // If this is a looped back M2 CMDU, we can treat is as handled successfully.
+    if (cmdu_rx.getClass<wfa_map::tlvApRadioIdentifier>())
+        return true;
```

Comment:
Same here, I think we don't need this special handling and we can just rely on the m1 check.

Reply:
Correct, removed.